controllers/tokenupload_controller.go

```diff
@@ -130,7 +103,8 @@ func (r *TokenUploadReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		}
 
 		logs.AuditLog(ctx).Info("manual token upload initiated", uploadSecret.Namespace, accessToken.Name)
-		err = r.TokenStorage.Store(ctx, &accessToken, &token)
+		// Upload Token, it will cause update SPIAccessToken State as well
+		err = r.TokenStorage.Store(ctx, accessToken, &token)
 		if err != nil {
 			logError(ctx, uploadSecret, fmt.Errorf("token storing failed: %w", err), r, lg)
```

Comment:
"failed to store the token: %w" would IMHO sound more natural...

Reply:
sure :)